B827EBFFFE569233.json

```diff
@@ -0,0 +1,19 @@
+{ 
+	"gateway_conf": 
+  { 
+		"gateway_ID": "B827EBFFFE569233", 
+		"servers": [ 
+			{ 
+				"server_address": "https://console.thethingsnetwork.org/gateways/b827ebfffe569233", 
```

Comment:
The server address must be one of the routers of TTN listed here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses